meta-lmp-base/recipes-graphics/wayland/weston_%.bbappend

```diff
@@ -0,0 +1,6 @@
+# Weston with Xwayland support (requires X11 and Wayland)
+PACKAGECONFIG[xwayland] = "-Dxwayland=true,-Dxwayland=false,libxcursor"
```

Comment:
OE already has the following:

`PACKAGECONFIG[xwayland] = "-Dxwayland=true,-Dxwayland=false"`

Why are we replicating this, for libxcursor?

Reply:
Not sure.  And TBH, we aren't enabling xwayland in this PR.  I can drop this for now and revisit later.